core/store/src/columns.rs

```diff
@@ -293,6 +293,13 @@ pub enum DBCol {
     /// Witnesses with the lowest index are garbage collected first.
     /// u64 -> LatestWitnessesKey
     LatestWitnessesByIndex,
+    /// A valid epoch sync proof that proves the transition from the genesis to some epoch,
+    /// beyond which we keep all headers in this node. Nodes bootstrapped via Epoch Sync will
+    /// have this column, which allows it to compute a more recent EpochSyncProof using block
+    /// headers collected after the stored EpochSyncProof.
+    /// - *Rows*: only one key with 0 bytes.
+    /// - *Column type*: `EpochSyncProof`
+    EpochSyncProof,
```

Comment:
Misc says:
```
    /// Column to persist pieces of miscellaneous small data. Should only be used to store
    /// constant or small (for example per-shard) amount of data.
```

EpochSyncProof can be 40+ MBs, I don't think this would be good for Misc.